examples/shadertoy_buffer.py

```diff
@@ -0,0 +1,91 @@
+from wgpu_shadertoy import Shadertoy
+from wgpu_shadertoy.inputs import ShadertoyChannelBuffer
+from wgpu_shadertoy.passes import BufferRenderPass
```

Comment:
You could consider exporting all of these objects from the root `wgpu_shadertoy` package just for convenience